docs/src/BrandingProvider.tsx

```diff
@@ -24,8 +26,9 @@ export default function BrandingProvider({ children, mode: modeProp }: BrandingP
   }, [mode]);
   return (
     <ThemeProvider theme={modeProp ? () => theme : theme}>
-      {!modeProp && <NextNProgressBar />}
-      {!modeProp && <CssBaseline />}
+      {modeProp ? null : <NextNProgressBar />}
+      {modeProp ? null : <CssBaseline />}
+      {modeProp ? null : <SkipLink />}
```

Comment:
So we get this:

<img width="521" alt="Screenshot 2022-07-30 at 16 17 11" src="https://user-images.githubusercontent.com/3165635/181918438-1367af79-062e-41f3-bac9-9b643e7843f4.png">